apps/builder/app/routes/$.tsx

```diff
@@ -9,6 +13,17 @@ import { ErrorMessage } from "~/shared/error";
 import { getBuildParams, dashboardPath } from "~/shared/router-utils";
 
 export const loader = async ({ request }: LoaderArgs) => {
+  const url = new URL(request.url);
+  // See remix.config.ts for the publicPath value
+  const publicPath = "/build/";
```

Comment:
shouldn't we get this value over some config or at least put it to some reusable place

Reply:
js and ts mix and moreover esm and cjs
between this file and remix.config.js

Looks like too many things to change is tsconfig, and usually causer runtime checks on exports as they are different in esm and cjs

Reply:
cc @TrySound do you know a simple way to share const between remix.config.js and routes/$.ts?

Reply:
I'm not sure. Some shit will happen for sure. Better hardcode. I'm sure this is temporary.